src/main/webapp/js/home-page.js

```diff
@@ -59,10 +59,21 @@ const homePage = {
   ],
 };
 
-document.addEventListener('DOMContentLoaded', () => {
-  createCarouselElements([2, 2, 3, 3], 4);
-  createHomePage(homePage);
-});
+/**
+ * Adds a function to window onload event.
+ * @param {function} func The function to be executed.
+ */
+function addLoadEvent(func) {
+  const oldonload = window.onload;
+  if (typeof oldonload == 'function') {
+    window.onload = function() {
+      oldonload();
+      func();
+    };
+  } else {
+    window.onload = func;
+  }
+}
```

Comment:
Use the function from https://github.com/googleinterns/step203-2020/blob/master/src/main/webapp/js/util.js?